test/fixtures/user/user.js

```diff
@@ -132,7 +132,7 @@ module.exports = () => {
       status: "active",
       roles: {
         member: true,
-        archivedMember: true,
+        archived: false,
```

Comment:
Why is value also changed with a key name?

Reply:
I wanted the value false for a test